cmd/state-remote-installer/main.go

```diff
@@ -95,6 +95,12 @@ func main() {
 		return
 	}
 
+	// Store sessionToken to config
+	err = cfg.Set(anaConst.CfgSessionToken, "remote_"+constants.RemoteInstallerVersion)
```

Comment:
Note: this will override any subsequent `ACTIVESTATE_SESSION_TOKEN` environment variables set by `install.sh` due to https://github.com/ActiveState/cli/blob/c6a5565c1eb184571bca3833bc8ccac7aac81eb8/cmd/state-installer/installer.go#L53-L57, but that should be addressed in DX-2228.